cfgov/v1/models/browse_page.py

```diff
@@ -52,8 +52,9 @@ class BrowsePage(CFGOVPage):
 
     template = 'browse-basic/index.html'
 
-    def get_page_js(self):
-        return super(BrowsePage, self).get_page_js() + ['secondary-navigation.js']
+    def get_page_js(self, js):
+        super(BrowsePage, self).get_page_js(js)
```

Comment:
If `get_page_js` is removed from `CFGovPage` (above), what is this calling/ how is this working?